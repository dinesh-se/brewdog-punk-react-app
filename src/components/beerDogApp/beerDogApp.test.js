import React from 'react';
import { shallow } from 'enzyme';
import BeerDogApp from './beerDogApp';

test('should render the app', () => {
  const wrapper = shallow(<BeerDogApp />);
  
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find('Header')).toHaveLength(1);
  expect(wrapper.find('BeerBanner')).toHaveLength(1);
  expect(wrapper.find('SearchForm')).toHaveLength(1);
  expect(wrapper.find('.footer-section').text()).toContain('Developed by');
});