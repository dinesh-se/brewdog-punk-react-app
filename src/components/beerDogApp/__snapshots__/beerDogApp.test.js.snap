// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the app 1`] = `
<div>
  <Header
    title="BrewDog"
  />
  <section
    className="container"
  >
    <BeerBanner />
    <SearchForm />
  </section>
  <footer
    className="footer-section"
  >
    <small>
      Developed by Dinesh Haribabu
    </small>
  </footer>
</div>
`;
